tests/carddatabase/filter_string_test.cpp

```diff
@@ -20,7 +20,7 @@ class CardQuery : public ::testing::Test
     {
         cat = CardDatabaseManager::getInstance()->getCardBySimpleName("Cat");
         notDeadAfterAll = CardDatabaseManager::getInstance()->getCardBySimpleName("Not Dead After All");
-        truth = CardDatabaseManager::getInstance()->getCardBySimpleName("Truth or Tale");
+        truth = CardDatabaseManager::getInstance()->getCardBySimpleName("Truth OR Tale");
```

Comment:
```suggestion
        truth = CardDatabaseManager::getInstance()->getCardBySimpleName("Truth");
```
I was intending the test case to test that “truth or tale” doesn’t match just “truth”